src/config/dynamic-theme-fixes.config

```diff
@@ -12125,20 +12125,30 @@ INVERT
 .add-more-forum
 .all-wraper
 .core_title_btns
+.day_rcmd > .class_title
+.fengchao-wrap-box span
 .focus_btn
+img[src*="sign_err.png"]
 .nav_wrap_add_border
 .save_face_bg
 .share_btn_wrapper
+.sub_nav_wrap
+.threadlist_rep_num
+.u-f-item
 
 CSS
+.card_banner {
+    opacity: 50%;
```

Comment:
Hmm interesting, what is this change for? We mostly don't need to poke around the opacity.

Reply:
Most banners are too bright for dark theme. Perhaps I should implement it in a different way?

Reply:
Well, we currently uses `filter: brightness(50%) sepia(40%) !important;` to ensure the brightness is down and add  bit of sepia over it to make it more natural.

Reply:
Looks like turning down some brightness is enough here (and looks the same as opacity). Updated.